schema.json

```diff
@@ -177,6 +177,42 @@
                 "required": ["traits"],
                 "additionalProperties": false
               }
+            },
+            "relations": {
+              "type": "array",
+              "description": "The relations array holds descriptions of ",
```

Comment:
[Fix] Presumably unfinished description.

Reply:
Lol, yes. Knew I had something to go back to. Fleshed out [here](https://github.com/OpenAssetIO/OpenAssetIO-Manager-BAL/pull/18/commits/e9ca6ac20dfe6b069ccb291e0276b2c1b1bb4830).